### PR TITLE
Merged Discord.bio presence & Discords.com presence together.

### PR DESCRIPTION
**Pull request by DiscordAnaxes**
Why? 

Discord.bio has been acquired by discords.com, and they have recently moved discord.bio to their website.

Proof of presence working:

### CODE REVIEW

**Comment by SlowLife1661** [comment]
ok wrong pr but well lgtm

**Review by rxri — CHANGES_REQUESTED** [review]
then delete discord.bio presence? xD

**Comment by DiscordAnaxes** [comment]
> then delete discord.bio presence? xD

Deleted 😅

File: websites/D/Discords.com/dist/metadata.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "category": "socials",
     "logo": "https://i.imgur.com/ZjtyAQx.png",
     "thumbnail": "https://i.imgur.com/LbR2uNX.png",
-    "version": "1.0.0",
+    "version": "1.0.1",
```
Comment (SlowLife1661) [inline websites/D/Discords.com/dist/metadata.json:11]: Doesn't really matter but it probably will make more sense if it's `1.1.0`?

Comment (DiscordAnaxes) [reply websites/D/Discords.com/dist/metadata.json:11]: Probably yeah. I'll change it.

File: websites/D/Discords.com/presence.ts [hunk 2]
```diff
@@ -195,14 +195,60 @@ presence.on("UpdateData", async () => {
...
   if (showTimestamp) presenceData.startTimestamp = browsingStamp;
 
-  if (presenceData.details == null) {
+  if (presenceData.details === null) {
     presence.setTrayTitle();
     presence.setActivity();
-  } else {
-    presence.setActivity(presenceData);
   }
+  presence.setActivity(presenceData);
```
Comment (SlowLife1661) [inline websites/D/Discords.com/presence.ts:253]: What is the point of this statement then

**Review by Slowlife01 — CHANGES_REQUESTED** [review]

File: websites/D/Discords.com/presence.ts (hunk 2 above)
Comment (ririxidev) [reply websites/D/Discords.com/presence.ts:253]: image

Comment (DiscordAnaxes) [reply websites/D/Discords.com/presence.ts:253]: lol

**Comment by DiscordAnaxes** [comment]
Additionally merging the Discord Templates (by EGGSY) presence into this one. Will make a new pr once completeted.

**Pull request closed without merging**